.circleci/config.yml

```diff
@@ -747,7 +747,7 @@ workflows:
           slack-webhook: '$SLACK_WOO'
     triggers:
       - schedule:
-          cron: '0 12,20 * * *'
+          cron: '0 12 * * *'
```

Comment:
@bsessions85 I'm not sure about this one - this change will also affect Woo canaries. Should I split jobs, so Woo remains at two runs per day?

Reply:
I would just run woo the same. It is still running on canary anyway